scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/ModelValidatorContext.java

```diff
@@ -120,6 +120,13 @@ public void updateTaskWithContext(Task task) {
                                                                     "java.lang.Boolean",
                                                                     "Double",
                                                                     "java.lang.Double",
+                                                                    "Long",
+                                                                    "java.lang.Longit g",
+                                                                    "Float",
+                                                                    "java.lang.Float",
+                                                                    "Math",
+                                                                    "java.lang.Math",
+                                                                    "org.codehaus.jackson.JsonNode",
```

Comment:
org.codehaus.jackson could be added in the prefixes (after you tested that it is usable in a practical example)